features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/extensions/publisher/assets/api/themes/default/js/api_subscriptions.js

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
This code is refactored in https://github.com/wso2/carbon-apimgt/commit/f72002eca74f242f41a3f259ab5cbc3cf7e8bc31